submissions/jeffersonMormah/css/contact.css

```diff
@@ -0,0 +1,243 @@
+body{
+    font-family: 'Franklin Gothic Medium';
+    background-color:white;
+}
+.nav-container{
+    width: 100%;
+    margin: auto;
+    overflow: hidden;
+    display: flex; 
+}
+ul {
+    padding: 0;
+    margin: 0; 
+}
+header {
+    background:rgb(250,128,114);
+    color: #F5F5DC;
+    padding-top: 10px;
+    min-height: 15%;
+    border-bottom:4px solid #0000FF;
+}
+header a {
+    color: #F5F5DC;
+    font-size: 17px;
+    font-weight: bold;
+    margin-left: 200px;
+}
+header li {
+    display: inline;
+    padding: 0 20px;
+}
+header .nav-title {
+    float: left;
+    padding-bottom: 12px;
+}
+header .nav-title h2,header .nav-title p{
+    margin:0;
+    padding:0;
+    color:white;
+}
+header nav{
+    float: right;
+    margin-top: 15px;
+}
+.contact-content{
+    overflow: hidden;
+    padding: 40px auto;
+    margin: auto;
+    width: 90%;
+}
+main {
+    float: left;
+    text-align: center;
+    width: 45%;
+    margin-top: 55px;
+}
+main h1 {
+    font-size: 30px;
+    text-align: center;
+    color: coral;
+}
+main h3 {
+    font-size: 20px;
+    color: blue;
+}
+.medialink{
+    float: left;
+}
+.medialink ul li {
+   margin-bottom: 8px;
+   font-size: 17px;
+   font-weight: bold;
+   display: inline-flex;
+   padding:10px ;
+   margin-left: 2px;
+   margin-top: -35px;
+}
+.medialink ul li img{
+    width: 70px;
+    text-align: center;
+    margin-top: 3px;
+}
+aside {
+    float: left;
+    width: 48%;
+    margin-top: 55px;
+    margin-left: 40px;  
+}
+aside .contact-form{
+    padding: 20px;
+    width: 94%;
+    border: 1px solid black;
+    border-radius: 20px;
+    margin-bottom: 12px;
+}
+
+aside .contact-form h1{
+    color: black;
+    font-size: 20px;
+    text-align: center;
+}
+aside .contact-form label {
+    color: rgb(250,128,114);
+    font-size: 20px;
+    margin-bottom: 20px;
+    display: inline-block;
+    width: 48%;
+    margin-left: 20px;
+}
+aside .contact-form input {
+    margin-top: 1px;
+    margin-left: 20px;
+    border: none;
+    border-bottom: 2px solid #000000;
+    width: 90%;
+    height: 30px;
+}
+aside .contact-form select {
+    margin-top: 1px;
+    margin-left: 20px;
+    border: none;
+    border-bottom: 2px solid #000000;
+    width: 90%;
+    height: 30px;
+}
+aside .contact-form textarea {
```

Comment:
This textarea is the only one on the page. Why not target `textarea` straight off without the other form of selection. @Jeffbuchi